packages/lab/src/combo-box-next/ComboBoxNext.tsx

```diff
@@ -247,7 +246,12 @@ export const ComboBoxNext = forwardRef(function ComboBoxNext<T>(
         role="combobox"
         variant={variant}
         value={inputValue}
-        {...getTriggerProps()}
+        ref={reference}
+        {...getReferenceProps}
+        {...getReferenceProps({
+          onBlur: () => blurHandler(),
```

Comment:
Are these the same effect compared to before? i.e. blur of input drives open to close?

Test with click on scrollbar of list